x/auth/ante/setup.go

```diff
@@ -74,7 +74,7 @@ func (sud SetUpContextDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 func SetGasMeter(simulate bool, ctx sdk.Context, gasLimit uint64) sdk.Context {
```

Comment:
Is there a plan to change the API here? Now simulate isn't used here.

Reply:
removed it, thanks for pointing it out